plugins/importer/src/main/java/com/managemc/importer/data/cache/VanillaPunishmentsCache.java

```diff
@@ -8,7 +8,7 @@
 public class VanillaPunishmentsCache {
 
   @Getter
-  private final PunishmentsCache baseCache;
+  protected final PunishmentsCache baseCache;
```

Comment:
Because EssentialsX and Vanilla share functionality (they both write to `banned-players.json` and `banned-ips.json`), we extend Vanilla's cache. You probably won't have to do this unless your plugin also borrows from Vanilla features.